core/arch/arm/plat-versal/conf.mk

```diff
@@ -0,0 +1,31 @@
+PLATFORM_FLAVOR ?= generic
+
+include core/arch/arm/cpu/cortex-armv8-0.mk
+
+$(call force,CFG_TEE_CORE_NB_CORE,2)
+$(call force,CFG_PL011,y)
+$(call force,CFG_GIC,y)
+$(call force,CFG_ARM_GICV3,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+
+# Disable core ASLR for two reasons:
+# 1. There is no source for ALSR seed, as ATF does not provide a
+#    DTB to OP-TEE. Hardware RNG is also not currently supported.
+# 2. OP-TEE does not boot with enabled CFG_CORE_ASLR.
```

Comment:
1 is fair enough but 2 is slightly worrying, any idea what may be going wrong?

Reply:
right, so the tool that generates the boot.bin (Xilinx's bootgen) cant find the entry point from the ELF file when this option is enabled. I could provide it at build time to TF-A  and that would get it running....I'll remove the second comment and expand.

Reply:
Alright no need to make it work (although it would be nice of course!), but yes please explain the limitation.